bedrock/mozorg/templates/mozorg/advertising.html

```diff
@@ -0,0 +1,103 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "base-protocol-mozilla.html" %}
+
+{% from "macros-protocol.html" import picto, call_out with context %}
+
+{% block page_title %}Advertise with Mozilla{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('protocol-call-out') }}
+  {{ css_bundle('mozilla-ads') }}
+{% endblock %}
+
+{% block content %}
+
+<header class="mzp-c-call-out mzp-t-dark ads-header">
+    <div class="mzp-l-content">
+      <h1>Find hard-to-reach audiences with Mozilla Ads</h1>
+      <p>Protecting your brand's safety and your customer's privacy</p>
+      <a href="https://mozilla.formstack.com/forms/advertising" class="mzp-c-button mzp-t-dark">Contact us</a>
+    </div>
+</header>
+
+<section class="ads-info mzp-l-content mzp-l-columns mzp-t-columns-two">
+  {% call picto(
+    base_el='div',
+    title='Hard-to-reach customers',
```

Comment:
**Question:** should we be adding dashes here?

Reply:
I checked with Emma and she said we could. Reason I chose to do this is because `hard to reach` in and of itself is an adjective describing the noun `customers`, and having hyphens between the words brings them together to emphasize that it's an adjective and is read as one, which makes it clearer to the reader, that these words aren't meant to be read as separate attributes to a sentence

Reply:
cool